sim/validator.py

```diff
@@ -41,31 +82,31 @@ def performance_test(
 
     # Add trailing slash
     btb_seq_path = os.path.join(btb_seq_path, '')
-    results_path = os.path.join(results_path, '')
+    output_path = os.path.join(output_path, '')
```

Comment:
the variable name `output_path` replaces `results_path` in most cases and is now used as the parent directory for the new `results_path` which is branch dependent, i.e. `output_path + '/' + branch_name`.

Reply:
Not sure about this, looking at it again. Why are we naming the results path according to the branch? Can't we just use the `output_path` for results?

Reply:
Less flexible to enforce the output path to be branch dependant.

Reply:
We can use `output_path` for all results, however, need a way to distinguish results generated on each branch. 
Could change the name of stats.csv to include the branch name. Also a sub-directory `site-stats\` is generated for each branch so would need to deal with this too.